images/protected-publish/oneshot_pod.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: access-node
+  namespace: pipeline
+  labels:
+    app: key-access
+spec:
+  selector:
+    matchLabels:
+      app: key-access
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: key-access
+    spec:
+      serviceAccountName: naccess
+      containers:
+      - name: access
+        image: ghcr.io/scottwittenburg/protected-publish:0.0.4
+        command: ["/bin/bash", "-c"]
+        args: ["cat /srcs/test_stacks.txt | /srcs/migrate.sh"]
```

Comment:
I'd rather specify the the list of stacks to publish when I launch the pod (i.e. when I `kubectl apply -f` these yamls files), but I haven't yet found a way to do that.

Reply:
@mvandenburgh in case you have an idea how to do this.

Reply:
Talked about making the args a list of mirror names.